### ISSUE TITLE
scheduled event failed

### ISSUE THREAD

**Issue by theowenyoung** [cloudflare/workers-rs#53]
`lib.rs`:

```rust
mod utils;
use worker::*;
// could not find `Schedule` in `worker`rustc(E0433)
#[event(scheduled)]
pub async fn main(ty: String, schedule: u64, cron: String) -> Result<()> {
    println!("test");
    Ok(())
}
```

Is there something I missed?

**Comment by nilslice** [comment]
No, you found some work in progress that is not supported yet. Sorry about that!

**Comment by theowenyoung** [comment]
Ok, I'll wait it until production!

**Comment by nilslice** [comment]
@mlafeldt - sorry about that. correct, as of now you can only use the `fetch` event. It shouldn't be a ton of work to get `scheduled` events working, but I just don't have the bandwidth at the moment to get it started. I'd be happy to point you in the right direction though if you're interested in taking a look at implementing it.

**Comment by zebp** [comment]
Fixed by #104
